files/en-us/mozilla/firefox/releases/90/index.html

```diff
@@ -65,6 +65,27 @@ <h4 id="DOM">DOM</h4>
   <li>Support for the <code>matrix</code> protocol has been added and can now be passed into the {{domxref('Navigator.registerProtocolHandler()')}} method as a valid scheme.</li>
 </ul>
 
+<h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
+<ul>
+  <li>Marionette now restricts to a single active WebDriver session ({{bug(1691047)}}).</li>
+  <li>Added support for the new type of user prompts in Firefox ({{bug(1686741)}})</li>
+  <li>Window handles make use of an unique id now, and don't change for process
```

Comment:
```suggestion
  <li>Window handles make use of a unique id now, and don't change for process
```
nit: an unique -> a unique